### PR TITLE
Win32 fixes

### PR DESCRIPTION
**Pull request by njeisecke**
Hi Alfred,

these are my fixes to make libre compile with VS 2017. It also disables memory tracking to avoid crashes due to missing mutex. Not sure if other parts would need a mutex too.

### CODE REVIEW

**Comment by Encamy** [comment]
Ty for updating project!
Instead of increasing FD_SETSIZE to 2048 i whould like to recommend use patch https://github.com/creytiv/re/commit/49834b7afd0fe31660108271ef7657a34caa6a6d
I can create pull request with it
@alfredh what about having two `main.c`? One for all platforms and one for windows
This patch was tested and it is working stably

**Comment by njeisecke** [comment]
@Encamy I have not tried that patch yet, but I've read some comments about the fd lookup being too slow.

**Comment by Encamy** [comment]
@njeisecke 
I didn't notice slowness of fd lookup. For example, it is able to make 200 concurrent call without any difficulties. It is stable and reliable

File: src/jbuf/jbuf.c [hunk 1]
```diff
@@ -89,8 +89,10 @@ static void frame_alloc(struct jbuf *jb, struct frame **f)
 		f0 = le->data;
 
 		STAT_INC(n_overflow);
+#if JBUF_STAT
```
Comment (alfredh) [inline src/jbuf/jbuf.c]: please remove this hunk

Comment (njeisecke) [reply src/jbuf/jbuf.c]: jbuf::stat is only available with JBUF_STAT defined (via RELEASE=1).

Comment (njeisecke) [reply src/jbuf/jbuf.c:92]: @alfredh Could you please have another look? Thanks!

**Pull request closed without merging**